src/fswAlgorithms/transDetermination/ephemDifferenceWithUncertainty/ephemDifferenceWithUncertainty.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ ISC License
+
+ Copyright (c) 2024, Laboratory for Atmospheric and Space Physics, University of Colorado at Boulder
```

Comment:
I've started putting 2024, University of Colorado at Boulder
worth confirming

Reply:
@patkenneally I thought we should use LASP?

Reply:
This line is the author. In this case that should be LASP, Uni of Co Bol... etc. University of Colorado is also correct just less specific maybe?

Reply:
Okay, I will leave it this way then.